testcases/crypto/dilithium_func.c

```diff
@@ -48,21 +48,18 @@ CK_RV run_SignVerifyDilithium(CK_SESSION_HANDLE session,
                               CK_OBJECT_HANDLE priv_key,
                               CK_OBJECT_HANDLE publ_key)
 {
-    CK_MECHANISM mech2;
+    CK_MECHANISM mech;
     CK_BYTE_PTR data = NULL, signature = NULL;
     CK_ULONG i, signaturelen;
     CK_MECHANISM_INFO mech_info;
     CK_RV rc;
 
-    testcase_begin("Starting with mechtype='%s', inputlen=%lu",
```

Comment:
Why is the testcase_begin removed?

Reply:
The testcase_begin macro prints some "----" on the console and I want to group the individual outputs of the different tests (generate, import, transport) including sign/verify within such output "groups".